Client/Includes/ClientEntityPool.hpp

```diff
@@ -18,7 +18,7 @@ class ClientEntityPool : public EntityPool {
 public :
     virtual void Draw(sf::RenderTexture &, TextureBag &);
 
-    virtual void AddEntity(std::string const &entityName, vec2<float> const &initialPos, TimeRef const & = TimeRef()) override;
+    virtual void AddEntity(std::string const &entityName, uint16_t, vec2<float> const &initialPos, TimeRef const &, std::initializer_list<void *> * = nullptr) override;
```

Comment:
Why are the poisitions unsigned ? How can we make an entity exit the screen without disappearing suddenly ?